Draw.it.Server/Program.cs

```diff
@@ -36,6 +38,14 @@
 builder.Services.AddSwaggerGen();
 builder.Services.AddApplicationServices().AddApplicationRepositories(builder.Configuration);
 
+// Optional: register EF Core if a Postgres connection string is present
+var connectionString = builder.Configuration.GetConnectionString("Postgres");
+if (!string.IsNullOrWhiteSpace(connectionString))
```

Comment:
```suggestion
// Register the DbContext if using DB repositories
var repoType = builder.Configuration.GetValue<string>("RepositoryType");
var connectionString = builder.Configuration.GetConnectionString("Postgres");
if (repoType == nameof(RepoType.Db) && !string.IsNullOrWhiteSpace(connectionString))
```

You have to rely on the repository type from the config instead of the presence of the connection string.

Also, don't forget to import `Enum`:
```cs
using Draw.it.Server.Enums;
```